web/src/components/profile/ProfileAddressLockedFields.test.tsx

```diff
@@ -17,17 +23,21 @@ function mockMaterialUI(): typeof materialUi {
 }
 
 describe("ProfileAddressLockedFields", () => {
+  beforeEach(() => {
+    useMockBusiness({});
+  });
+
   const renderComponent = ({
     address,
     municipalities,
   }: {
-    address?: Address;
+    address?: FormationAddress;
```

Comment:
@AnJuHyppolite I'm not going to tag each instance in the PR, but my rec here would be that any place we use the type `FormationAddress` the corresponding variable name should probably reference `formationAddress`.